src/Microsoft.Data.SqlClient/src/Microsoft/Data/Common/DbBatchCommand.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Data;
+using System.Data.Common;
+
+namespace Microsoft.Data.Common
+{
+    /// <include file='../../../../doc/snippets/Microsoft.Data.Common/DbBatchCommand.xml' path='docs/members[@name="DbBatchCommand"]/DbBatchCommand/*' />
+    public abstract class DbBatchCommand
```

Comment:
Same as previous comments, I think this becomes internal-only for netfx, right?

Reply:
see above